Simplenote/build.gradle

```diff
@@ -85,6 +92,10 @@ dependencies {
     implementation 'com.automattic:tracks:1.2'
     implementation 'org.wordpress:passcodelock:2.0.2'
 
+    // Kotlin and KTX
+    implementation 'androidx.core:core-ktx:1.3.2'
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
```

Comment:
Question (❓): With the newest `Kotlin` plugin version, the need to explicitly define a dependency is removed, as such wouldn't it be okay if this dependency is removed altogether?

Reply:
That is true. I double check and I removed it.

Reply:
Great, thanks! 🙏